index.html

```diff
@@ -0,0 +1,128 @@
+                    </div>
+                    <div id="myLinks" class="textLinks menuText myFont">
+                        <a id="closeHamburger" href="javascript:void(0);" class="icon left">
+                            <img id="closeMenuImage" src="images/xMenu_50.png" alt="Close Menu">
+                        </a>
+                        <a id="home" href="#home" tabindex="1">Home</a>
+                        <a id="about" href="#about" tabindex="1">About</a>
+                        <a href="#Program" tabindex="2">Program</a>
+                        <a href="#Join" tabindex="3">Join</a>
+                        <a href="#Sponsor" tabindex="4">Sponsor</a>
+                        <a href="#News" tabindex="5">News</a>
+                    </div>
+                    <div class="importantLink">
+                        <a href="#contact" tabindex="5">CC Campaign</a>
+                    </div>
+                </div>
+            </div>
+        </header>
+        <section>
+            <div id="main" class="generalPadding main">
+                <!--Background image-->
+                <div  id="mainBackground" class="mainBackground"></div>
+                <!--Menu Hamburger-->
+                <div id="menuHamburgerContainer" class="menuHamburgerContainer">
+                    <a id="menuHamburger" href="javascript:void(0);" class="menuHamburger">
+                        <img id="baraMenuImage" src="images/barMenu.png" alt="Hamburger Menu" class="">
+                    </a>
+                </div>
+                <div class="mainContainer myFont">
+                    <h2>"Hello! Sharing world"</h2>
+                    <h1>CREATIVE COMMONS<br>GLOBAL SUMMIT<br>2015</h1>
+                    <p class="darkerColor">A joyful celebration believing in the value of openess and sharing, creating a positive change with people from over 80 countries is taking place in October, in Korea</p>
+                    <h3 class="darkerColor">2015.10.15&#40;THU&#41; ~ 16&#40;FRI&#41;</h3>
+                    <H4>&#64; National Museum Korea, Art Center Nabi and more</H4>
+                </div>
+            </div>
+        </section>
+        <section class="kinds generalPadding myFont">
+            <div class="kindsTitles">
+                <h1>Main Program</h1>
+                <hr>
+            </div>
+            <div id="kindsContainer" class="kindsContainer">
+
+            </div>
+            <div class="kindsLkn myFont">
+                <a href="javascript:void(0);">SEE THE WHOLE PROGRAM</a>
+            </div>
+            <div class="kindsBtn myFont">
+                <a href="javascript:void(0);" class="">Join CC Global summit 2015</a>
+            </div>
+        </section>
+        <section class="bands generalPadding myFont">
+            <div class="bandsTitles">
+                <h1>Featured Music Bands</h1>
+                <hr>
+            </div>
+            <div class="bandsContainer" id="bandsContainer">
+
+            </div>
```

Comment:
- [ ] [REQUIRED] Good job so far! Please give `chess`(e.g If you want you use every kind of color instead of `chess`, not only white 😄) shape to `background` of each `avatar` pictures 🙏🏼 

![image](https://github.com/alexansaa/ConcertAdv/assets/63915024/15f916d7-4034-49bc-bb38-106f46711651)